variables.tf

```diff
@@ -843,6 +855,30 @@ variable "wait_till" {
   }
 }
 
+variable "kube_version" {
+  description = "The version of the OpenShift cluster that should be provisioned (format 4.x). This is only used during initial cluster provisioning, but ignored for future updates. If no value is passed, or the string 'latest' is passed, the current latest OCP version will be used."
+  type        = string
+  default     = null
+  validation {
```

Comment:
Don't we need to continue to support classic IKS? In which case, I think that OCP versions need to be passed like this: `"4.x_openshift"`
And classic IKS will be passed like `1.18`

We should discuss